test/logger_test.exs

```diff
@@ -10,7 +10,7 @@ defmodule Plexy.LoggerTest do
     end)
 
     assert logged =~ "foo=1"
-    assert logged =~ "test=\"bar\""
+    assert logged =~ "test=bar"
```

Comment:
That's a good question. I think so (See: https://github.com/heroku/engineering-docs/blob/24a38ae630c897710e78c565d17f53c20b2daa02/guides/logs-as-data.md#events-annotations )